python-threatexchange/threatexchange/signal_type/text_md5.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Wrapper around the Text MD5 signal types.
+"""
+
+import typing as t
+import hashlib
+import pathlib
+from threatexchange.content_type.content_base import ContentType
+from threatexchange.content_type.text import TextContent
+
+from threatexchange.signal_type import signal_base
+from threatexchange import common
+from threatexchange.signal_type.raw_text import RawTextSignal
+from threatexchange.fetcher.apis.fb_threatexchange_signal import (
+    HasFbThreatExchangeIndicatorType,
+)
+
+
+class TextMD5Signal(
+    signal_base.SimpleSignalType,
+    signal_base.TextHasher,
+    signal_base.FileHasher,
+    HasFbThreatExchangeIndicatorType,
+):
+    """
+    Simple signal type for Text MD5s.
+    """
+
+    INDICATOR_TYPE = "TEXT_URL_MD5"
```

Comment:
nit: remove URL 
Maybe "HASH_TEXT_MD5" to correspond with `INDICATOR_TYPE = "HASH_URL_MD5"` in url_md5.py ?